x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -113,7 +113,20 @@ export const FormContext: React.FC<Props> = ({
         : null,
     [children, connectors, isLoadingConnectors]
   );
-  return <Form form={form}>{childrenWithExtraProp}</Form>;
+  return (
+    <Form
+      onKeyDown={(e: KeyboardEvent) => {
```

Comment:
I am not proud of it. But I couldn't come up with a better solution.

Reply:
The `Tags` and `Description` fields (still) behave as-expected when they have focus and `Enter` is pressed.

Given that this view is implemented as a `<form>`, it would be reasonable to expect `Enter` to submit the form when the `Tags` and `Description` fields don't have focus, but to keep this PR (targeted to `7.15.0`) laser-focused, stopping propagation of the event without adding additional conditions here feels appropriate.